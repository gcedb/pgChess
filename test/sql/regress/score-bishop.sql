\set my_game 'k7/8/8/2p5/8/4B3/8/7K w - - 0 2'
\set pg2podg_iterations 1
\set pg2podg_depth_target 1

\ir _reset-games.sql
\ir _.sql
