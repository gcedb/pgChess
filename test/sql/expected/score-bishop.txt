psql:test/sql/regress/_.sql:51: INFO:  1 turns to go (1 games)
ID: k7/8/8/2p5/8/4B3/8/7K w - - 0 2
psql:test/sql/regress/_.sql:51: INFO:  Score: -4.200
Choices:
	1.400 {} k7/8/8/2B5/8/8/8/7K b - - 0 2
	0.500 {} k7/8/8/2p5/8/4B3/6K1/8 b - - 1 2
	0.300 {} k7/8/8/2p5/5B2/8/8/7K b - - 1 2
	0.200 {} k7/8/8/2p5/8/4B3/7K/8 b - - 1 2
	0.100 {} k7/8/8/2p5/3B4/8/8/7K b - - 1 2
	0.100 {} k7/8/8/2p5/8/4B3/8/6K1 b - - 1 2
	0.000 {} k7/8/8/2p3B1/8/8/8/7K b - - 1 2
	0.000 {} k7/8/8/2p5/8/8/3B4/7K b - - 1 2
	-0.200 {} k7/8/7B/2p5/8/8/8/7K b - - 1 2
	-0.200 {} k7/8/8/2p5/8/8/5B2/7K b - - 1 2
	-0.200 {} k7/8/8/2p5/8/8/8/2B4K b - - 1 2
	-0.500 {} k7/8/8/2p5/8/8/8/6BK b - - 1 2
psql:test/sql/regress/_.sql:51: INFO:  0 turns to go (13 games)
ID: k7/8/8/2B5/8/8/8/7K b - - 0 2
psql:test/sql/regress/_.sql:51: INFO:  game ended after 1 iterations
